addon/styles/_frost-numeric-select.scss

```diff
@@ -0,0 +1,99 @@
+$height: 30px;
+$border: 0.3px solid $frost-color-input-border;
+$focus-border: 1px solid $frost-color-blue-1 !global;
+$frost-color-button-hover-bg: rgba($frost-color-blue-1, .1) !global;
+$arrow-focus-color: $frost-color-blue-1 !global;
+$error-text-color: $frost-color-input-error-border;
+$disabled-border: 1px solid $frost-color-input-disabled-border;
+$disabled-bg: $frost-color-input-disabled-bg;
```

Comment:
I'd like to see these variables prefixed since they're going to end up in the global namespace - so

`$frost-number-select-height: 30px;` etc.